### PR TITLE
Use UUID for error codes for Form validator.

### PR DESCRIPTION
**Pull request by Koc**
| Q | A |
| --- | --- |
| Branch? | 2.8 |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | tiny, but it allowed http://symfony.com/doc/current/contributing/code/bc.html#changing-classes |
| Deprecations? | no |
| Tests pass? | I hope so :) |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/Form/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+2.8.12
+-----
+
+ * [BC BREAK] Changed values of the Symfony\Component\Form\Extension\Validator\Constraints\Form constants.
```
Comment (fabpot) [inline src/Symfony/Component/Form/CHANGELOG.md]: We cannot have a BC break in a patch release. Is it really a BC break? People MUST use the constant anyway, not the value.

Comment (Koc) [inline src/Symfony/Component/Form/CHANGELOG.md]: Ok, so can I just remove `[BC BREAK]` from the changelog?

**Comment by fabpot** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @Koc.

**Comment by Koc** [comment]
@fabpot I've choosen 2.8 branch because uuid codes was added in 2.8, not in 2.7

**Comment by Koc** [comment]
See #15154 for details

**Comment by xabbuh** [comment]
I agree with @Koc. We should revert the merge on the `2.7` branch and merge it into `2.8` afterwards.

**Comment by fabpot** [comment]
Oops, sorry about that. Going to fix that now.

**Comment by fabpot** [comment]
Thank you @Koc.

**Pull request closed without merging**